pkg/remotewrite/config.go

```diff
@@ -1,108 +1,85 @@
 package remotewrite
 
 import (
+	"crypto/tls"
 	"encoding/json"
-	"net/url"
+	"net/http"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/grafana/xk6-output-prometheus-remote/pkg/remote"
 	"github.com/kubernetes/helm/pkg/strvals"
-	promConfig "github.com/prometheus/common/config"
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/storage/remote"
+
 	"go.k6.io/k6/lib/types"
 	"gopkg.in/guregu/null.v3"
 )
 
 const (
-	defaultPrometheusTimeout = time.Minute
-	defaultFlushPeriod       = 5 * time.Second
-	defaultMetricPrefix      = "k6_"
+	defaultURL          = "http://localhost:9090/api/v1/write"
+	defaultTimeout      = 5 * time.Second
+	defaultPushInterval = 5 * time.Second
+	defaultMetricPrefix = "k6_"
 )
 
 type Config struct {
-	Mapping null.String `json:"mapping" envconfig:"K6_PROMETHEUS_MAPPING"`
-
-	Url null.String `json:"url" envconfig:"K6_PROMETHEUS_REMOTE_URL"` // here, in the name of env variable, we assume that we won't need to distinguish between remote write URL vs remote read URL
-
+	// here, in the name of env variable, we assume that we won't need to distinguish between remote write URL vs remote read URL
+	URL     null.String       `json:"url" envconfig:"K6_PROMETHEUS_REMOTE_URL"`
 	Headers map[string]string `json:"headers" envconfig:"K6_PROMETHEUS_HEADERS"`
 
 	InsecureSkipTLSVerify null.Bool   `json:"insecureSkipTLSVerify" envconfig:"K6_PROMETHEUS_INSECURE_SKIP_TLS_VERIFY"`
-	CACert                null.String `json:"caCertFile" envconfig:"K6_CA_CERT_FILE"`
+	CACert                null.String `json:"caCertFile" envconfig:"K6_PROMETHEUS_CA_CERT_FILE"`
```

Comment:
Isn't that from another pull request? :thinking:

Reply:
I expect we will merge the config PR soon so we will get this rid when I will rebase, just keep this open as a reminder.

Reply:
Yeah, I was confused that this PR has only 1 commit :sweat_smile: